packages/azure-kusto-data/src/security.ts

```diff
@@ -14,21 +14,24 @@ export class AadHelper {
         }
 
         if (!!kcsb.aadUserId && !!kcsb.password) {
-            this.tokenProvider = new TokenProvider.UserPassTokenProvider(kcsb.dataSource, kcsb.aadUserId, kcsb.password, kcsb.authorityId);
+            this.tokenProvider = new TokenProvider.UserPassTokenProvider(kcsb.dataSource, kcsb.aadUserId, kcsb.password, kcsb.authorityId, kcsb.timeoutMs);
```

Comment:
does this still work if kcsb.timeoutMs is not provided?

Reply:
sure

Reply:
its an optional param in all